src/site/markdown/server/manage-webadmin.md

```diff
@@ -2496,6 +2497,45 @@ Response codes:
 
 Not implemented yet.
 
+## Deleted Messages Vault
+
+Deleted messages from an user will be stored in a restricted place where only administrators can access.
+This place is called the Deleted Messages Vault.
+
+ - [Restore Deleted Messages](#Restore_deleted_messages)
+
+### Restore Deleted Messages
+
+An user can request an administrator to restore his deleted messages that are kept into the Deleted Message Vault.
```

Comment:
s/An user/a user/ ?

Reply:
IMO who calls the endpoint + how the admin and the user interact together is clearly out of scope.

We should focus on what this endpoint does.